### DISCUSSION TITLE
SUD dependency tree for a given sentence

### DISCUSSION THREAD

**Discussion by yannis1962** [surfacesyntacticud/website#47] [Q&A] [answered]
Hi,

I'm writing a textbook and would like to include a brief presentation of SUD. I would like to include an example of SUD tree, for the same sentence I use also for UD, namely

The fact that Alice waited surprised Bob.

I tried to install the conversion tool on my machine, but it was impossible. Could you please send me the SUD syntax tree for this sentence? Thank you so much!

**Comment by perrier54** [comment, marked as answer]
The attached Conll file [example.txt](https://github.com/surfacesyntacticud/guidelines/files/11191463/example.txt) and its graphic representation below.

![example](https://user-images.githubusercontent.com/29424639/230931632-5c738ce8-0647-4ae0-8c8b-25f5c2387e0e.svg)

**Comment by yannis1962** [comment]
Thank you so much for your prompt reply!

> Le 10 avr. 2023 à 17:23, perrier54 ***@***.***> a écrit :
> 
> 
> The attached Conll file example.txt <https://github.com/surfacesyntacticud/guidelines/files/11191463/example.txt> and its graphic representation below.
> 
>  <https://user-images.githubusercontent.com/29424639/230931632-5c738ce8-0647-4ae0-8c8b-25f5c2387e0e.svg>
> —
> Reply to this email directly, view it on GitHub <https://github.com/orgs/surfacesyntacticud/discussions/47#discussioncomment-5571602>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAFXC7NHTEXPVJTNPA6BOJ3XAQQVTANCNFSM6AAAAAAWZCGTBA>.
> You are receiving this because you authored the thread.
> 

 <http://www.imt-atlantique.fr/>	Yannis HARALAMBOUS
Professor
Computer Science Department
UMR CNRS 6285 Lab-STICC
 <https://www.imt-atlantique.fr/en/person/yannis-haralambous> <https://twitter.com/y_haralambous> <https://www.linkedin.com/in/yannis-haralambous-5529073?trk=hp-identity-name>Technopôle Brest-Iroise CS 83818
29238 Brest Cedex 3, France
Une École de l'IMT <http://www.imt.fr/>

‌UNIX Book Units have all of the virtues and none of the vices of fitment furniture.
They are as modern as movies and as classical as Greece.
‌(Brian Kernighan, ‌UNIX: A History and a Memoir)‌
